config/samples/metallb.io_v1alpha1_addresspool.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample1
+  namespace: metallb-system
+spec:
+  address-pool:
+    name: default
+    protocol: layer2
+    addresses:
+    - 172.18.0.100-172.18.0.200
+
+---
+
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample2
+  namespace: metallb-system
+spec:
+  address-pool:
+    name: gold
+    protocol: layer2
+    addresses:
+    - 172.20.0.100-172.20.0.200
+    auto-assign: false
+
+---
+
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample3
+  namespace: metallb-system
+spec:
+  address-pool:
+    name: silver
+    protocol: layer2
+    addresses:
+    - 172.21.0.100-172.21.0.200
+    avoid-buggy-ips: true
```

Comment:
I think I would prefer removing this option completely.  I'd like to see if anyone really needs this.  We could always consider adding it in the future, and it's possible to express the same thing without the option, though with a more tedious definition of address ranges potentially.

Reply:
I know u don't like that option so its optional and has the default of false so instead of changing the API if we found out somehow a user needs this , its easy to remove but since its the metallb config doc there should be someone using it ?

Reply:
removed